en-US/firefox/welcome/page2.lang

```diff
@@ -0,0 +1,49 @@
+## NOTE: Demo https://bedrock-demo-craigcook.oregon-b.moz.works/firefox/welcome/2/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/welcome/2/
+
+
+# HTML page title
+# "Pocket" is a product brand name and shouldn't be translated
+;Pocket - Save news, videos, stories and more
+Pocket - Save news, videos, stories and more
+
+
+;Your time online is worth protecting
+Your time online is worth protecting
+
+
+# "Pocket" is a product brand name and shouldn't be translated.
+;Discover and save stories in Pocket — and come back to them when you’re free.
+Discover and save stories in Pocket — and come back to them when you’re free.
+
+
+# "Pocket" is a product brand name and shouldn't be translated.
+;Pocket is built right into Firefox, so you can easily save stories as you find them, then read them later on any device.
+Pocket is built right into Firefox, so you can easily save stories as you find them, then read them later on any device.
+
+
+# "Pocket" is a product brand name and shouldn't be translated.
+;Activate Pocket
+Activate Pocket
+
+
+;Save content from everywhere
+Save content from everywhere
+
+
+;Grab articles, videos, and links from any website by clicking the Pocket icon in your browser toolbar.
```

Comment:
Pocket is mentioned in most of the strings in this page. Do you want to add another comment for this Pocket? :).

Reply:
Just to be thorough!